src/test/java/posters/pageobjects/pages/checkout/PlaceOrderPage.java

```diff
@@ -37,9 +37,10 @@
 
     @Override
     @Step("ensure this is a place order page")
-    public void isExpectedPage()
+    public PlaceOrderPage isExpectedPage()
     {
         headline.should(exist);
```

Comment:
Please call `super.isExpectedPage();` at first in case we would add some default functionality.

Reply:
@ufjena this hasn't been covered yet.